### PR TITLE
Improve neutron configuration.

### PR DESCRIPTION
**Pull request by ss7pro**

### CODE REVIEW

File: neutron/templates/configmaps/neutron.conf.yaml.j2 [hunk 1]
```diff
@@ -62,10 +60,11 @@ data:
     [agent]
     root_helper = sudo /var/lib/kolla/venv/bin/neutron-rootwrap /etc/neutron/rootwrap.conf
 
+    transport_url = rabbit://{{ rabbitmq.admin_user }}:{{ rabbitmq.admin_password }}@{{ rabbitmq.address }}:{{ rabbitmq.port }}
```
Comment (DTadrzak) [inline neutron/templates/configmaps/neutron.conf.yaml.j2]: You should add to neutron/manifest.jsonnet rabbitmq.port variable.

Comment (DTadrzak) [inline neutron/templates/configmaps/neutron.conf.yaml.j2]: Wrong section, move it to default

File: neutron/templates/configmaps/neutron.conf.yaml.j2 [hunk 2]
```diff
@@ -33,6 +28,15 @@ data:
...
+    max_l3_agents_per_router = 3
+    l3_ha_network_type = vxlan
+
+    dhcp_agents_per_network = 3
+
+    network_auto_schedule = True
+    router_auto_schedule = True
 
     [nova]
     auth_url = {{ keystone.auth_url }}
```
Comment (ss7pro) [inline neutron/templates/configmaps/neutron.conf.yaml.j2]: DONE

File: neutron/templates/configmaps/neutron.conf.yaml.j2 (hunk 1 above)
Comment (ss7pro) [inline neutron/templates/configmaps/neutron.conf.yaml.j2]: DONE

Comment (ss7pro) [inline neutron/templates/configmaps/neutron.conf.yaml.j2]: DONE

**Comment by DTadrzak** [comment]
LGTM

**Pull request merged**